.github/workflows/auto-label-issue.yml

```diff
@@ -0,0 +1,19 @@
+name: Auto Label issues
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Label issues
+        uses: andymckay/labeler@5c59dabdfd4dd5bd9c6e6d255b01b9d764af4414
+        with:
+          add-labels: "triage"
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          
```

Comment:
Remove this very last empty line.

End result should be just a single empty line at the very end of the file.